mapswipe_workers/Dockerfile

```diff
@@ -12,10 +12,12 @@ RUN mkdir -p /usr/local/mapswipe_workers/
 RUN mkdir -p /var/log/mapswipe_workers/
 RUN mkdir -p /var/lib/mapswipe_workers/
 RUN mkdir -p /var/lib/mapswipe_workers/api-data
-RUN mkdir -p /var/lib/mapswipe_workers/api-data/agg_res_by_task_id/
-RUN mkdir -p /var/lib/mapswipe_workers/api-data/agg_res_by_project_id_and_date/
-RUN mkdir -p /var/lib/mapswipe_workers/api-data/agg_progress_by_project_id_and_date/
-RUN mkdir -p /var/lib/mapswipe_workers/api-data/agg_res_by_user_id_and_date/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/agg_results/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/groups/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/history/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/projects/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/results/
+RUN mkdir -p /var/lib/mapswipe_workers/api-data/tasks/
```

Comment:
I would try to use a variable for these repeated paths, it helps avoid typos, and also helps confirm that you indeed want `/var/log` and not `/var/lib` on the first line, it helps confirm that the line below should indeed by `/usr/local`, and so on.